catalog/core/catalog-core-actions/src/main/java/org/codice/ddf/catalog/actions/AbstractMetacardActionProvider.java

```diff
@@ -111,7 +111,8 @@ public String getId() {
      * {@inheritDoc}
      */
     public <T> boolean canHandle(T subject) {
-        return (subject instanceof Metacard) && (canHandleMetacard((Metacard) subject));
+        return (subject instanceof Metacard) && ((Metacard) subject).getTags()
```

Comment:
I put it here because all of the action providers that extend this can't handle anything but resource tagged metacards. Ideally each of the handlers would do a better job of checking if they can handle a metacard and this wouldn't be needed. There is another ticket out there to look at improvements to our action providers that could incorporate this kind of change.